pkg/resources/testdata/TestAcc_GrantApplicationRole/application/test.tf

```diff
@@ -1,3 +1,8 @@
+provider "snowflake" {
+  profile   = "default"
+  warehouse = "\"${var.warehouse}\""
+}
+
```

Comment:
Remove the providers from tests as they are not needed

Reply:
Removed. But without this custom config, these tests are failing on my machine. Let's see what happens on CI.

Reply:
You're right, it works now. Also, I fixed application deps.